src/layouts/Main/Header.tsx

```diff
@@ -0,0 +1,112 @@
+import './index.scss';
+import { NavLink } from 'react-router-dom';
+import { Button } from 'antd';
+import { PlusOutlined } from '@ant-design/icons';
+import LogoSmall from 'components/HomePage/icons/LogoSmall';
+import routeConstants from 'route/routeConstant';
+import { memo, useState, FC } from 'react';
+import Logout from './Logout';
+import Searchbar from 'layouts/Main/Searchbar';
+
+const items = [
+    {
+        id: 1,
+        name: 'TRANG CHỦ',
+        url: routeConstants.MAINPAGE,
+    },
+    {
+        id: 2,
+        name: 'BÀI VIẾT',
+        url: routeConstants.POST,
+    },
+];
+interface Props {
+    firstName: string;
+}
+// eslint-disable-next-line no-undef
+const Header: FC<Props> = memo(({ firstName }) => {
+    const [isDropdownOpen, setIsDropdownOpen] = useState(false);
+    if (!firstName) {
+        firstName = 'default';
```

Comment:
[Assignment to function parameter 'firstName'](https://eslint.org/docs/latest/rules/no-param-reassign)